workflows/templates/distributed-regrid.yaml

```diff
@@ -231,11 +222,6 @@ spec:
           sel_time = os.environ["SELTIME"]
           regrid_method = os.environ["REGRID_METHOD"]
           domain_file = os.environ["DOMAIN_FILE"]
-          # Space-delim string of variables and coordinates that do use "time" dimension.
-          # If data has something like `lat_b` or `height`, and you care about getting it in output,
-          # those names need to be in this variable!
-          # TODO: We could make this dynamically find non-time dependent variables/coord variables.
```

Comment:
ah. I hadn't seen that TODO !

Reply:
They're everywhere and I forget them all the time, too. 👍